src/app/globals.css

```
@@ -4,7 +4,7 @@
 
 @import '../styles/base.css';
 @import '../styles/utilities.css';
-@import '../styles/article.css';
+@import '../styles/blog.css';
```

Stylelint (17.5.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)